### DISCUSSION TITLE
Want to update this bug！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！！

### DISCUSSION THREAD

**Discussion by c913168497** [unit-mesh/auto-dev#171] [Q&A] [answered]
![image](https://github.com/unit-mesh/auto-dev/assets/23418458/4dfe3787-e787-492a-9c93-7fc75975c495)
这个类的这行代码改成 这样的话，模板保存的 提示词 会在提问的时候更新，否则，由于私有变量的原因被缓存起来
![image](https://github.com/unit-mesh/auto-dev/assets/23418458/ba7d8728-1d39-4fce-8dce-c9c7d799071d)

**Comment by phodal** [comment, marked as answer]
直接来  PR ？

**Reply by c913168497** [reply]
那个，我基于这个开源版本，新增了 功能特性，可以对选择的内容，进行一次 正则过滤，做初级内容筛洗，目前阶段，选择的内容有时候太多了，但是让我一个个去选中又太麻烦了 
![image](https://github.com/unit-mesh/auto-dev/assets/23418458/14025bc4-a4a5-408b-ba1b-1750bc6fe3cb)
由于这个我想基于团队来使用，我希望能建立一个公共的提示词，模板库，用于下载，就像搭建一个 APIfox 那样的，团队API平台，统一标准

**Reply by c913168497** [reply]
还有，就是基于单机版本 IDEA ，其实提示词分析能力比较差，对于一些代码的内容筛洗，比较弱，我好希望能建议一个 服务，在发送给 模型前，先在我们的服务中，抓取远程gitlb代码，进行一步初级代码内容上下文，抓取，并筛洗的流程后，整理一个不错的提示词，再发给模型，让模型能生成更准确 更优秀的 回答，您看我的建议好吗？

**Reply by phodal** [reply]
> 那个，我基于这个开源版本，新增了 功能特性，可以对选择的内容，进行一次 正则过滤，做初级内容筛洗，目前阶段，选择的内容有时候太多了，但是让我一个个去选中又太麻烦了 
> 
> ![image](https://github.com/unit-mesh/auto-dev/assets/23418458/14025bc4-a4a5-408b-ba1b-1750bc6fe3cb)
> 
> 由于这个我想基于团队来使用，我希望能建立一个公共的提示词，模板库，用于下载，就像搭建一个 APIfox 那样的，团队API平台，统一标准

这个也欢迎来 PR 哈哈哈

**Reply by phodal** [reply]
> 还有，就是基于单机版本 IDEA ，其实提示词分析能力比较差，对于一些代码的内容筛洗，比较弱，我好希望能建议一个 服务，在发送给 模型前，先在我们的服务中，抓取远程gitlb代码，进行

理论上就是 自定义 agent？

**Reply by c913168497** [reply]
对，我可以pr吗？

**Reply by phodal** [reply]
欢迎来  PR ～

**Reply by c913168497** [reply]
哈哈哈～好～
